meshroom/core/node.py

```diff
@@ -1210,6 +1213,7 @@ def has3DOutputAttribute(self):
     globalExecMode = Property(str, globalExecMode.fget, notify=globalExecModeChanged)
     isExternal = Property(bool, isExtern, notify=globalExecModeChanged)
     isComputed = Property(bool, _isComputed, notify=globalStatusChanged)
+    isComputable = Property(bool, lambda self: self.getGlobalStatus() != Status.INPUT, notify=globalStatusChanged)
```

Comment:
Correct but I'd rather have the lambda in a separate function instead, like we're doing for all the other properties.

```
def _isComputable(self):
    return self.getGlobalStatus() != Status.INPUT

...

isComputable = Property(bool, _isComputable, notify=globalStatusChanged)
```